vote/cnab/app/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
This file doesn't appear to be used (instead, the `Makefile` runs the appropriate actions); therefore, I'd suggest we remove it.